scripts/globals/mobskills/severing_fang.lua

```diff
@@ -0,0 +1,37 @@
+-----------------------------------
+--  Severing Fang
+--
+--  Description: Bites at all targets in front. Additional effect: Defense Down
+--  Type: Physical
+--  Utsusemi/Blink absorb: 2 shadows
+--  Range: Front arc (cone)
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    -- Do not use this weapon skill on targets behind. Sub-Zero Smash
+    -- should trigger in this case.
+    if target:isBehind(mob) then
+        return 1
+    end
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 2
+    local accmod = 1
+    local dmgmod = 1.5
+    local info = MobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, TP_NO_EFFECT)
+    local dmg = MobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.PIERCING, MOBPARAM_WIPE_SHADOWS)
```

Comment:
Header says: `--  Utsusemi/Blink absorb: 2 shadows`

Reply:
Looks like that aligns with the data recorded for the ability (in that when hit with it, it'll absorb two shadows)

Reply:
The comment is correct. This was a copy/paste error from the script I branched off of.

Good eye!

It should be `info.hitslanded` instead of `MOBPARAM_WIPE_SHADOWS`.

Reply:
Corrected in the latest commit.